certified-connectors/CognitiveServicesSpeechToText/Readme.md

```diff
@@ -0,0 +1,28 @@
+## Azure Cognitive Services Batch Speech-to-text Connector
```

Comment:
@RichardSunMS please follow our template for the readme file. you can find an example [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md) and for raw view for code [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md). Please include all required sections.

Reply:
Thanks and updated.